app/components/header/account-dropdown.hbs

```diff
@@ -23,7 +23,6 @@
       {{else}}
         <Header::AccountDropdownLink @text="Subscribe" @icon="credit-card" {{on "click" (fn this.handleSubscribeClick dd.actions)}} />
       {{/if}}
-
```

Comment:
@Arpan-206 make sure to review and remove unrelated changes from a PR - we usually do keep empty lines between blocks where indentation changes for readability